### DISCUSSION TITLE
Search Results Link Open in new tab

### DISCUSSION THREAD

**Discussion by azizsayfiddin** [philc/vimium#4587] [General]
When I'm in search mode with query and enter the search result with link, it goes in the current tab, but I want it to open in new tab. Is there something LinksHints.OpenInNewTab thing?

**Comment by philg-dev** [comment]
Could you check out if the discussion here helps? It sounds to me like you've got the same issue as the person in #4562 a little while ago.

Details in comment:
https://github.com/philc/vimium/issues/4562#issuecomment-2466211527
